pom.xml

```diff
@@ -25,6 +25,16 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>2.20.0</version>
+        </dependency>
```

Comment:
do we need 2 dependencies or only logj4-core?

Reply:
Yes, we can use only logj4-core) My bad